src/plugins/esProxyAuth/index.js

```diff
@@ -0,0 +1,46 @@
+let _ = require('lodash');
+let btoa = require('btoa');
```

Comment:
Not sure if btoa does something special, but base64 encoding is supported natively
```js
let btoa = str => (new Buffer(str, 'utf8')).toString('base64')
```